doc/user/vrrp.rst

```diff
@@ -490,9 +490,10 @@ execute the following command:
    and IPv6 addresses - are used to create a VRRP router on their parent
    interfaces, with VRRP IPvX addresses taken from the addresses assigned to
    the macvlan devices. The generated configuration appears in the output of
-   ``show run``, which can then be modified as needed and written to the config
-   file. The ``version`` parameter controls the protocol version; if using
-   VRRPv2, keep in mind that IPv6 is not supported and will not be configured.
+   ``show run | include vrrp``, which can then be modified as needed and
+   written to the config file. The ``version`` parameter controls the protocol
+   version; if using VRRPv2, keep in mind that IPv6 is not supported and will
+   not be configured.
```

Comment:
I think this bit is fine as it is, this sort of implies you need the `| include` there in order to see VRRP config.